web-console/4.0-designs/binding/binding.md

```diff
@@ -0,0 +1,30 @@
+# Binding
+
+## Create binding
+![](img/binding-prompt.png)
+- New service instances will have a section at the top that describe what service bindings are for and how to use them. The box will also contain a button to create a service binding for the service instance.
+- This box should disappear once it has been dismissed or a binding has been created.
```

Comment:
Some services aren't bindable. We should probably check and only show the box if binding is allowed. 

Unfortunately the logic for figuring it out is somewhat complicated :(

Reply:
What happens in the current UI? Is the binding section shown regardless?

Reply:
Right now, we aren't checking for `bindable`, so the binding section is shown. We have a bug tracking:

https://bugzilla.redhat.com/show_bug.cgi?id=1640509